packages/server/tests/acceptance/htsPrecompile.spec.ts

```diff
@@ -214,6 +226,22 @@ describe('HTS Precompile Acceptance Tests', async function () {
       const { approved } = (await tx.wait()).events.filter(e => e.event === 'ApprovedAddress')[0].args;
       expect(approved).to.equal('0x0000000000000000000000000000000000000000');
     });
+
+    it('should be able to transfer nft with transferFrom', async function() {
+      expect(await NFTokenContract.balanceOf(accounts[0].wallet.address)).to.equal(1);
+      expect(await NFTokenContract.balanceOf(accounts[2].wallet.address)).to.equal(0);
+
+      // grant KYC
+      const grantKycTx = await baseHTSContractOwner.grantTokenKycPublic(NftHTSTokenContractAddress, accounts[2].wallet.address, { gasLimit: 1_000_000 });
+      const responseCodeGrantKyc = (await grantKycTx.wait()).events.filter(e => e.event === 'ResponseCode')[0].args.responseCode;
+      expect(responseCodeGrantKyc).to.equal(TX_SUCCESS_CODE);
+
+      //transfer
+      await NFTokenContract.transferFrom(accounts[0].wallet.address, accounts[2].wallet.address, NftSerialNumber);
```

Comment:
there was no approval prior to this transferFrom

Reply:
Working on it.